test/unit/commands/hello.spec.ts

```diff
@@ -1,6 +1,8 @@
 import {expect, test} from '@oclif/test';
 
-describe('hello', () => {
+describe('hello', function () {
+  this.slow(3000);
```

Comment:
Why do we need 3s/5s for the no-op hello command?

Reply:
I added this because on my computer the hello test occasionally timesout. I am unsure as to exactly why this happen though. The slow method was added because I didn't want to see the red (ms) after I ran the unit tests. Is there a reason for us keeping the hello command and hello tests around or can we just delete the files?